### DISCUSSION TITLE
How to remove log box

### DISCUSSION THREAD

**Discussion by felislynx-silae** [ExperienceLovelace/ha-floorplan#241] [Q&A] [answered]
I'm using latest version of ha-floorplan.
After setup i have working floorplan but under it there is huge div with log output. I can remove logs, but div is still showing with "Clear logs" button. How to remove it?

**Comment by OzGav** [comment, marked as answer]
You don’t show your yaml but you likely have the following so comment it out or remove it:
`log_level: info`

**Comment by felislynx-silae** [comment]
Thank you. I had empty log level :)
